modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -21,174 +21,36 @@ endif::[]
-
-* `/v2/models/<model-name>/infer`
-
-*vLLM ServingRuntime for KServe*
-
-* `:443/version`
-* `:443/docs`
-* `:443/v1/models`
-* `:443/v1/chat/completions`
-* `:443/v1/completions`
-* `:443/v1/embeddings`
-+
-NOTE: The vLLM runtime is compatible with the OpenAI REST API. For a list of models that the vLLM runtime supports, see link:https://docs.vllm.ai/en/latest/models/supported_models.html[Supported models].
-+
-NOTE: To use the embeddings inference endpoint in vLLM, you must use an embeddings model that the vLLM supports. You cannot use the embeddings endpoint with generative models. For more information, see link:https://github.com/vllm-project/vllm/pull/3734[Supported embeddings models in vLLM].
-+
-
-As indicated by the paths shown, the single-model serving platform uses the HTTPS port of your OpenShift router (usually port 443) to serve external API requests.
---
-
-. Use the endpoint to make API requests to your deployed model, as shown in the following example commands:
-+
-NOTE: If you enabled token authorization when deploying the model, add the `Authorization` header and specify a token value.
+. Use the endpoint to make API requests to your deployed model.
 
 ifdef::upstream[]
-+
---
-*Caikit TGIS ServingRuntime for KServe*
-[source,subs="+quotes"]
-----
-curl --json '{"model_id": "<model_name>", "inputs": "<text>"}' \
-https://<inference_endpoint_url>:443/api/v1/task/server-streaming-text-generation \
--H 'Authorization: Bearer <token>'
-----
-
-*Caikit Standalone ServingRuntime for KServe*
-
-.REST
-[source]
-----
-curl -H 'Content-Type: application/json' -d '{"inputs": "<text>", "model_id": "<model_id>"}' <inference_endpoint_url>/api/v1/task/embedding -H 'Authorization: Bearer <token>'
-----
-
-.gRPC
-[source]
-----
-grpcurl -insecure -d '{"text": "<text>"}' -H \"mm-model-id: <model_id>\" <inference_endpoint_url>:443 caikit.runtime.Nlp.NlpService/EmbeddingTaskPredict -H 'Authorization: Bearer <token>'
-----
-
-*TGIS Standalone ServingRuntime for KServe*
-[source]
-----
-grpcurl -proto text-generation-inference/proto/generation.proto -d \
-'{"requests": [{"text":"<text>"}]}' \
--insecure <inference_endpoint_url>:443 fmaas.GenerationService/Generate \
--H 'Authorization: Bearer <token>'
-----
-
-*OpenVINO Model Server*
-[source]
-----
-curl -ks <inference_endpoint_url>/v2/models/<model_name>/infer -d \
-'{ "model_name": "<model_name>", \
-"inputs": [{ "name": "<name_of_model_input>", "shape": [<shape>], "datatype": "<data_type>", "data": [<data>] }]}' \
--H 'Authorization: Bearer <token>'
-----
-
-*vLLM ServingRuntime for KServe*
-[source]
-----
-curl -v https://<inference_endpoint_url>:443/v1/chat/completions -H \
-"Content-Type: application/json" -d '{ \
-"messages": [{ \
-"role": "<role>", \
-"content": "<content>" \
-}] -H 'Authorization: Bearer <token>'
-----
---
+[NOTE]
+====
+For a list of sample commands, see link:{odhdocshome}/serving-models/#inference-endpoints_serving-large-models[Inference endpoints]
```

Comment:
Missing period (.) at the end of the sentence, after the link.

Reply:
```suggestion
For a list of sample commands, see link:{odhdocshome}/serving-models/#inference-endpoints_serving-large-models[Inference endpoints].
```